st2actions/st2actions/scheduler/handler.py

```diff
@@ -50,7 +50,6 @@
 # (< 5 seconds). If an item is still being marked as processing it likely indicates that the
 # scheduler process which was processing that item crashed or similar so we need to mark it as
 # "handling=False" so some other scheduler process can pick it up.
```

Comment:
You removed the constant but left so precious code comment describing the `EXECUTION_SCHEDUELING_TIMEOUT_THRESHOLD_MS` with no owner.

Please move it to a more relevant place.

Reply:
Can you please explain exactly what you require here.  I am not following.  Thanks.

Reply:
I would appreciate a suggested change here.  Thanks.

Reply:
@guzzijones Check the 
https://github.com/StackStorm/st2/blob/f5e56acf841d5fac6f7aed2df897a00eccb4bbad/st2actions/st2actions/scheduler/handler.py#L47-L53

The code comment was explaining how `EXECUTION_SCHEDUELING_TIMEOUT_THRESHOLD_MS` constant was used.
This PR removed the `EXECUTION_SCHEDUELING_TIMEOUT_THRESHOLD_MS` constant, but left the code comment.

It would be better to move this useful code comment to the other place to keep that precious information.

Reply:
Should I move this down near the member variable in the class?
``` self._execution_scheduling_timeout_threshold_ms```

Reply:
Sure, that sounds like a perfect place to me :+1:

Reply:
@guzzijones I think once this last thing is done we can merge this.